### ISSUE TITLE
Documents need improvement

### ISSUE THREAD

**Issue by skawaguchi-flipp** [TykTechnologies/tyk-sync#116]
Between the [README](https://github.com/TykTechnologies/tyk-sync/blob/master/README.md) and the [Tyk docs](https://tyk.io/docs/tyk-sync/#usage), there are several gaps that are serious barriers to usage:

1. There is no API definition for the `.tyk.json` file. How am I supposed to create one? What are the attributes? If they exist in another API def, please point me to it. This doc should be the single source of truth.

2. There is no explanation for how to support multiple APIs. I'm assuming it would be to create folders with just a `.tyk.json` in it like `/api-definitions/api1/.tyk.json` and `/api-definition/api2/.tyk.json`. After some digging around, I found this structure and I assume adding another file would be how I support multiple APIs:
    ```json
    {
        "type": "oas",   
        "files": [
            {
                "file": "./curator.json",
                "oas": {
                    "version_name": "v1" 
                }
            }
        ]
    }
    ```
    fixing Item 1 above would fix this.

3. The examples could be improved. One with the docker version referencing local files, and another where you're referring to a private GitHub repo and need to authenticate are the big ones I could see.

**Comment by skawaguchi-flipp** [comment]
I've posted on the community forum as well: https://community.tyk.io/t/tyk-sync-example/3652/6?u=skawaguchi_flipp

**Comment by oluwaseyeayinla** [comment]
@skawaguchi-flipp Yes, the documentation may need some improving. I found going through the source could reveal some more details about the structure. I have already replied to the community post but I can also drop links where on what helped me
- https://github.com/TykTechnologies/tyk-sync/blob/5ceb2539a8c533e455fdec1e8e728ed012bc55e1/tyk-vcs/spec.go
- https://github.com/TykTechnologies/tyk-sync/blob/5ceb2539a8c533e455fdec1e8e728ed012bc55e1/tyk-vcs/getter.go#L201-L253
- https://github.com/lonelycode/integration-test
